charts/gardener-metrics-exporter/charts/application/templates/_helpers.tpl

```diff
@@ -0,0 +1 @@
+../../../templates/_helpers.tpl
```

Comment:
Symbolic link

Reply:
Regarding the symbolic links I followed the way it is handled in other repositories. Example for g/g controlplane components [ref](https://github.com/gardener/gardener/blob/master/charts/gardener/controlplane/charts/runtime/values.yaml). I do not have strong opinion of how should we do it in this repository. As another example I can give the [oidc-webhook repository](https://github.com/gardener/oidc-webhook-authenticator/tree/master/charts/oidc-webhook-authenticator) where I have used global values + actual different `values.yaml` files for the two subcharts. This has its advantages but it also makes it a little bit harder to maintain.

More on the helm subcharts and global values topic can be found [here](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/).